frontend/src/app/modules/projects/create-project-modal/create-project-modal.component.spec.ts

```diff
@@ -0,0 +1,27 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
```

Comment:
it seems we can remove it

https://trello.com/c/xvI5HRQ5/61-as-a-developer-i-want-to-have-all-fe-tests-removed